apps/server/src/modules/learnroom/learnroom-api.module.ts

```diff
@@ -20,7 +21,7 @@ import {
 } from './uc';
 
 @Module({
-	imports: [AuthorizationModule, LessonModule, CopyHelperModule, LearnroomModule],
+	imports: [AuthorizationModule, LessonModule, CopyHelperModule, LearnroomModule, AuthorizationReferenceModule],
```

Comment:
Hier braucht es das AuthorizationReferenceModule nicht, oder? In den beiden UCs wird doch nur der normale AuthorizationService benutzt.

Reply:
courseExportUC verwenden ihn noch. Ich denke da kann man ihn wie an anderen Stelle auch noch ersetzten. 
CourseCopyUC das gleiche

Ich denke das sollte ich noch machen. Ich habe nur Angst mehr und mehr in merge Konflikte zu laufen wenn ich ihn jetzt bis zur nächsten "dev" Time in 8 Tagen liegen lasse. :P 

Andererseits gibt es auch noch ein Problem in sharing-create-token.api und tool-configuration.api die aber die gleiche Ursache haben könnten.

Reply:
BC - 5436